lib/src/generators/contract/openapi3.ts

```diff
@@ -45,25 +46,36 @@ export function openApiV3(contractDefinition: ContractDefinition): OpenApiV3 {
           operationId: endpoint.name,
           description: endpoint.description,
           tags: endpoint.tags,
-          parameters: getParameters(endpoint),
+          parameters: getParameters(contractDefinition.types, endpoint),
           ...(endpoint.request.body && {
             requestBody: {
               content: {
                 "application/json": {
-                  schema: openApi3TypeSchema(endpoint.request.body.type)
+                  schema: openApi3TypeSchema(
+                    contractDefinition.types,
+                    endpoint.request.body.type
+                  )
                 }
               },
               description: endpoint.description || ""
             }
           }),
           responses: {
             ...(endpoint.defaultResponse
-              ? { default: response(endpoint.defaultResponse) }
+              ? {
+                  default: response(
+                    contractDefinition.types,
+                    endpoint.defaultResponse
+                  )
+                }
               : {}),
             ...endpoint.responses.reduce<{
               [statusCode: string]: OpenAPIV3Body;
             }>((acc, responseNode) => {
-              acc[responseNode.status.toString(10)] = response(responseNode);
+              acc[responseNode.status.toString(10)] = response(
```

Comment:
Where does the '10' come from?

Reply:
It's the base in which to represent the number. It actually defaults to base 10, so I could remove it if you think that's more readable.

Reply:
ohh never knew this, TIL. Thanks!